foundations/foundation-common/src/test/java/org/apache/servicecomb/foundation/common/http/TestHttpUtils.java

```diff
@@ -55,6 +55,21 @@ public void parseParamFromHeaderValue_keyNotFound() {
     Assertions.assertNull(HttpUtils.parseParamFromHeaderValue("xx;k=", "kk"));
   }
 
+  @Test
+  public void parseParamFromHeaderValue_valueWithQuotationMark() {
+    Assertions.assertEquals("v", HttpUtils.parseParamFromHeaderValue("xx;k=\"v\"", "k"));
+  }
+
+  @Test
+  public void parseParamFromHeaderValue_valueWithBlank() {
+    Assertions.assertEquals("v", HttpUtils.parseParamFromHeaderValue("xx; k=v", "k"));
+  }
+
+  @Test
+  public void parseParamFromHeaderValue_valueWithQuotationMarkAndBlank() {
+    Assertions.assertEquals("v", HttpUtils.parseParamFromHeaderValue("xx;k= \"v\"", "k"));
+  }
```

Comment:
Maybe the following cases is enough 

* key1=a;key2=b
* key1=a; key2=b
* key1="a";key2="b"
* key1="a"; key2="b"
* key1="a"; key2="b c.txt"